packages/nimble-components/src/table/specs/table-column-specs/table-column-button-hld.md

```diff
@@ -0,0 +1,155 @@
+# Button Table Column
+
+NOTE: No longer planned (at least, with this limited scope). Client use cases that would have used this form of the button column are served well enough by an action menu item.
```

Comment:
I added this note since we're no longer planning to create this column type (in this proposed, limited scope). I think we should still submit this as a starting point/history in case we revisit the idea of a button column in the future. @jattasNI if you agree and approve, I'll just move it out of draft and submit.

Reply:
Any objection to keeping your branch around indefinitely? If not, I think I'd rather link to the branch from the related issue and leave it unsubmitted, similar to an innovation day research project. Or maybe copy the contents of the HLD into a comment in the issue if you're worried about the branch disappearing.

I'd personally vote against having unapproved HLDs be committed to the repo since they'd be potential sources of confusion (even with a disclaimer at the top) and if we ever revive them I'd rather we look at the proposal with fresh eyes, using our up-to-date templates, than look at a diff versus a previously unapproved proposal.

Reply:
Ok, I don't feel that strongly. I'll just leave the branch on the remote. Even if it ends up deleted somehow, there aren't really any critical insights in this doc--mostly just some mechanical stuff we'd have to rewrite.